src/connect-zbt-2/getting-started-with-thread/switching-from-one-thread-adapter-to-another-thread-adapter.md

```diff
@@ -0,0 +1,72 @@
+---
+zendesk:
+  article_id: 32087461954589
+  name: Migrating from one Thread adapter to another Thread adapter
+  position: 30
+  labels: connect-zbt-2, getting-started
+productName: "Home Assistant Connect&nbsp;ZBT-2"
+---
+
+If you already have a Thread adapter running but want to switch to a {{ productName }}, follow these steps:
+
+## Prerequisites
+
+{% steps %}
+{% prereq %}
+{% image "/static/img/connect-zbt-2/connect-zbt-2-unboxed-02.png" "{{ productName }} with USB extension cable in packaging" %}
+{% stepContent %}
+
+- [Home Assistant Operating System](https://www.home-assistant.io/docs/glossary/#home-assistant-operating-system) installed on your smart home hub. For example:
+  - on a Home Assistant Green, where Home Assistant OS is preinstalled
+  - on a Home Assistant Yellow or on a Raspberry Pi
+- A Thread network and a Thread adapter connected to your Home Assistant hub
+- Latest updates installed
+- {{ productName }} adapter and a USB extension cable
+- Thread devices
+
+{% endstepContent %}
+{% endprereq %}
+{% endsteps %}
+
+## To enable dedicated Thread support
```

Comment:
Should this title be changed to something like "To migrate an existing Thread network to Connect ZBT-2", similar to the Zigbee articles?